oscm-initdb/Dockerfile

```diff
@@ -7,7 +7,34 @@ ENV DB_HOST_APP=oscm-db DB_PORT_APP=5432 DB_NAME_APP=bssapp DB_USER_APP=bssappus
 ENV SQL_DUMP_GLOBALS=globals.sql SQL_DUMP_BSS=bss.sql SQL_DUMP_BSSJMS=bssjms.sql SQL_DUMP_BSSAPP=bssapp.sql
 
 ENV AUTH_MODE=INTERNAL HOST_FQDN=localhost LOG_LEVEL=INFO
+ENV LOG_FILE_PATH=../logs
+ENV LOG_CONFIG_FILE=./log4j.properties
 ENV REPORT_ENGINEURL=https://${HOST_FQDN}:8681/birt/frameset?__report=\${reportname}.rptdesign&SessionId=\${sessionid}&__locale=\${locale}&WSDLURL=\${wsdlurl}&SOAPEndPoint=\${soapendpoint}&wsname=Report&wsport=ReportPort
+ENV MP_ERROR_REDIRECT_HTTPS=https://${HOST_FQDN}:8081/oscm-portal/marketplace?mid=8f78f156
+
+ENV DECIMAL_PLACES=2
+ENV MAIL_JA_CHARSET=UTF-8
```

Comment:
The corresponding setting in the template file still has a literal value; it must be set to the variable placeholder in order to work.
Also the setting is commented out in the template.